doc/ant-xgboost_user_guide.md

```diff
@@ -0,0 +1,265 @@
+### _user guide:_ Ant-XGBoost on sqlflow
+
+#### Overview
+
+[Ant-XGBoost](https://github.com/alipay/ant-xgboost) is fork repo of [dmlc/xgboost](https://github.com/dmlc/xgboost), which is maintained by active contributors of dmlc/xgboost in Alipay.inc.
+
+Ant-XGBoost is built on top of dmlc/xgboost but with additional support on kubernetes and automatic training. 
+
+#### Why Ant-XGBoost
+
+##### Generalized Early Stopping Strategy
+In dmlc/xgboost, early stopping is applied when there is no improvement after consecutive `n` boosting rounds. 
```

Comment:
`dmlc/xgboost` stops when no significant improvements in the recent n boosting rounds, where n is a configurable parameter.